src/components/Calculator.jsx

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+
+const Screen = ({ result }) => <span className="screen">{result}</span>;
+Screen.propTypes = { result: PropTypes.number.isRequired };
+
+const Oprand = ({ className, data }) => (
+  <button type="button" className={className}>
+    {data}
+  </button>
+);
+Oprand.propTypes = { className: PropTypes.string.isRequired };
+Oprand.propTypes = { data: PropTypes.string.isRequired };
+
+const Number = ({ className, value }) => (
+  <button type="button" className={className}>
+    {value}
+  </button>
+);
+Number.propTypes = { className: PropTypes.string.isRequired };
+Number.propTypes = { value: PropTypes.number.isRequired };
+
+// Calculator
+/* eslint-disable */
```

Comment:
- [ ] Good job converting this to a class-based component as suggested by the previous reviewer, but i can see you disabled this linter. Kindly note that linters force you to write clean and good code, so please make sure you fix your linter issues rather than disabling them 😄 

for this linter issue, I will suggest you add a constructor and an empty state. This way, the error will disappear as you have converted the component to a stateful component 👍🏽 


e.g 

```
  constructor(props) {
    super(props);
    this.state = {};
  }
```